.eslintrc.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  extends: '@netlify/eslint-config-node',
+}
```

Comment:
This might be good to specify also `overrides`. `overrides` are not inherited unless directly imported, which means when someone adds `overrides`, they might get confused since they would lose the `overrides` from `@netlify/eslint-config-node`, making linting fail.

```js
const { overrides } = require('@netlify/eslint-config-node')

module.exports = {
  extends: '@netlify/eslint-config-node',
  overrides: [...overrides],
}
```